app/src/components/navigation/ProfileMenu.vue

```diff
@@ -39,8 +78,15 @@ const userNavigation: {
                 <span
                     class="ml-4 text-sm font-semibold leading-6 text-zinc-900 dark:text-zinc-50"
                     aria-hidden="true"
+                    v-if="isAuthenticated"
                     >{{ user?.name }}</span
                 >
+                <span
+                    class="ml-4 text-sm font-semibold leading-6 text-zinc-900 dark:text-zinc-50"
+                    aria-hidden="true"
+                    v-else
+                    >Profile</span
```

Comment:
Just a small thing and can be easily changed later, but perhaps "Profile" is a bit strange if you're not logged in. Maybe it should just be `Menu`? Can quickly check at next standup.